src/api/integration/wallet_test.go

```diff
@@ -562,7 +562,7 @@ func TestVerifyWallet(t *testing.T) {
 
 	// check with incorrect seed
 	isBip, err = c.VerifySeed("nut ")
-	testutil.RequireError(t, err, "seed is not a valid bip39 seed")
+	require.Equal(t, bip39.ErrSurroundingWhitespace, err)
```

Comment:
The `err` returned by `c.VerifySeed` is a different error type.

Reply:
Fixed; test had failed